runtime/runtime_test.go

```diff
@@ -2377,7 +2376,14 @@ func TestRuntimeCyclicImport(t *testing.T) {
 
 	require.Error(t, err)
 	require.IsType(t, Error{}, err)
-	assert.IsType(t, ast.CyclicImportsError{}, err.(Error).Unwrap())
+
+	// Temporary to help catch checking error
+	ee := err.(Error).Unwrap()
+
+	require.IsType(t, &ExtendedParsingCheckingError{}, ee)
+	eee := ee.(*ExtendedParsingCheckingError)
+
+	assert.IsType(t, ast.CyclicImportsError{}, eee.Unwrap())
```

Comment:
The old could should have used `utils.RequireErrorAs` here, and not manual unwrap. This would have prevented you to have to add the additional manually unwrapping step.

```suggestion
	utils.RequireErrorAs(t, err, &ast.CyclicImportsError{})
```

Then again, `ast.CyclicImportsError` is gone in `master` (it is now `sema.CyclicImportsError`), are you sure you are working off of `master`?

Reply:
As discussed, it was branched off `0.9.2` so resolving comment.